src/ctype/CTypeSchema.ts

```diff
@@ -133,7 +133,46 @@ export const CTypeWrapperModel = {
       type: 'string',
       minLength: 1,
     },
-    metamodel: {
+  },
+  required: ['schema'],
+}
+
+export const CTypeWrapperMetadata = {
```

Comment:
Let's come up with a better naming here. It is not wrapping the ctype, it is rather the model or json-schema for the metadata + ctypehash object

Reply:
I changed it to the following `CTypeMetadataModel`